indico/web/client/js/react/components/ICSCalendarLink.jsx

```diff
@@ -56,6 +57,7 @@ const ICSExportOptions = ({options, selected, handleSetOption}) => (
               primary={key === selected}
               label={idx === 0 ? Translate.string('Export') : undefined}
               labelPosition={idx === 0 ? 'left' : undefined}
+              styleName={idx === options.length - 1 ? 'final-button' : 'mid-button'}
```

Comment:
Instead of conditionally changing the styleName, since it's only the first labeled button having an incorrect right radius, I would suggest setting `styleName="button-group"` to the parent `Button.Group` and then:

```
.button-group :global(.ui.labeled.button) button:global(.ui.button):last-child {
  border-top-right-radius: 0;
  border-bottom-right-radius: 0;
}
```

I think we could also keep this CSS in `ICSCalendarLink` and get rid of `ICSExportOptions` since they are both in the same file.

Reply:
Thanks, fixed. Also deleted the file :+1: